detect_secrets/audit/common.py

```diff
@@ -44,7 +45,7 @@ def open_file(filename: str) -> 'LineGetter':
 def get_raw_secret_from_file(
     secret: PotentialSecret,
     line_getter_factory: Callable[[str], 'LineGetter'] = open_file,
-) -> str:
+) -> Any:
```

Comment:
Uh, is there a reason why this is `Any`? `str` looks sound to me.

Reply:
Hi @domanchi, we made this change because the mypy recommended it. I'm going to check it right now. When I have something else I will reply this comment.

Reply:
Oh, I think that I got it... We should change `str` to `Optional[str]'. I will test again, run the pre-commit hook and push this change. What do you think @domanchi?

Reply:
Ohhh, I see the issue. It's because of https://github.com/Yelp/detect-secrets/blob/master/detect_secrets/core/potential_secret.py#L66, so `mypy` (rightly) expects this to return the same type.

Yeah, `Optional[str]` works for me.